packages/terra-tabs/src/_CollapsibleTabs.jsx

```diff
@@ -118,9 +118,9 @@ class CollapsibleTabs extends React.Component {
     let isMenuHidden = true;
     for (let i = 0; i < this.props.children.length; i += 1) {
       const tab = this.container.children[i];
-      const tabMarginLeft = parseFloat(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-left'));
-      const tabMarginRight = parseFloat(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-right'));
-      const minWidth = parseFloat(window.getComputedStyle(tab, null).getPropertyValue('min-width'));
+      const tabMarginLeft = parseInt(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-left'), 10);
```

Comment:
@avinashg1994 Just FYI, this was recently updated to use float to address an issue related to overflow - https://github.com/cerner/terra-framework/pull/822

@supreethmr

Reply:
Yes, I was aware of that. I just reverted those changes in this PR to check if they were the reason behind the inconsistent wdio test failures for Responsive Tabs.